### DISCUSSION TITLE
IslandChestValues Update Suggestion

### DISCUSSION THREAD

**Discussion by djtmk1** [BG-Software-LLC/SuperiorSkyblock2#1361] [Denied Feature Requests]
So I have become very interested in this add-on. I am implanting it onto my Skyblock, but have run into a few issues. 

Issues: 
- Players can place any block in there, even if it does not give a value. 
- You can not prevent players from glitching the GUI menu and duping the island level. 
- Once a player puts a value block into the /is chest GUI, they can take it back out. 

Suggested Fixes: 
- Make this add-on configurable, so we can set it to where only certain blocks can be added. 
- Make it to where once a block or a stack of a block is placed into the chest, they can not move it around the GUI to dupe their levels. 
- Don't allow players to remove these blocks out of the GUI by hand, but maybe from a command like /is chest withdraw or something. 

Suggested add-ons: 
- In a config, allow us to set Valueblocks for these chests. This would also allow us to only let certain blocks or items to go into the chests. (As suggested in the above fixes) 
- Allow us to run a CMD for a physical unbreakable chest, so players can choose between a GUI and or a physical chest. Players would have to run a CMD to get this chest and then place it on their island. Once placed, it can not be removed.

**Comment by OmerBenGera** [comment]
Thanks for the detailed suggestion.
In my opinion, its too specific for your server needs, and I dont think it will be uses by everyone. Therefore, I recommend you to fork the addon github and apply the changes for yourself :)
